magefile.go

```diff
@@ -126,6 +128,7 @@ func init() {
 
 	// Extra dependencies must be specified here to avoid a circular dependency.
 	packageImages[pkoPackageName].ExtraDeps = []interface{}{Generate.PackageOperatorPackage}
+	packageImages[remotePhasePackageName].ExtraDeps = []interface{}{Generate.RemotePhasePackage}
```

Comment:
Lines 64-66 say "// Note that you can't reference the Generate mage target in ExtraDeps
	// since that would result in a circulat dependency. They must be added via init() for now." If that comment is out of date could you remove it?

Reply:
Let's revisit the whole magefile next year, I think most of the comments are still alright.